submissions/best-of-its-genre-nanogamer7/origins.md

```diff
@@ -0,0 +1,46 @@
+## Best of its genre - Nanogamer7
+
+We've all heard of Factorios insanely [good ratings](https://steamdb.info/stats/gameratings/) - consistently rising to second place after Portal 2 again and again.[^1] Obviously many others will try to imitate this success in the automation genre, but where did Factorio get inspired from? That's what I'll be trying to answer in this first article of hopefully a multi-part series.
+
+### Automation defined
+
+The first difficult question we'll have to answer though, would be what automation even is. In Cities:Skylines for example you are building your city, which then (theoretically) functions autonomously. The problem here is, your job isn't to automate - the cars drive themselves, once you build roads - but to manage it. In Minecraft on the other hand, you are the one to make the automation even possible, just with pistons and hoppers, instead of inserters and belts. Comparing these two examples, I think you can loosely define the automation genre as *"making stuff that makes stuff"*.
```

Comment:
> cars drive themselves

Belts move items by themselves. There isn't much difference between placing a train in CS than in Factorio, except Factorio has much more customization. I agree City Skylines isn't automation as Factorio, but you could try to define this difference better. Maybe CS has automation more as a side-thing with more focus on aesthetics. It has very limited automation options.

Or at least try to connect CS to stuff-that-make-stuff

Reply:
Hmm, good point there. I'd argue the difference between streets and belts would be at junctions, where in CS each car decides by themselves where they need to go, while in Factorio the items need to be sorted by groups/categories by a player created setup (splitters). I'll see what I can think of